src/main/java/com/vailsys/freeclimb/api/call/CallOptions.java

```diff
@@ -27,14 +27,20 @@
 	 */
 	private Integer timeout;
 	/**
-	 * The callId of the call that created this call (if one exists).
+	 * The callId of the call that created this call (if one exists). Required if no `applicationId` or `callConnectUrl` has been provided.
```

Comment:
Seems like the other comments in this file are trying to keep an 80 character line limit in comments for readability, so this line should be broken up to fit that standard.